### ISSUE TITLE
TriMet feed includes instructions to operators

### ISSUE THREAD

**Issue by downeymj** [kona314/transit-fedilerts#9]
The TriMet feed seems to include messages to operators which are not particularly well suited to regular consumers of alerts, and mostly end up just adding to noise and generating so many messages it's hard to follow the feed for its AP account. 

I would expect the types of messages posted to this feed to more or less match those types of messages that are passed through TriMet's Twitter feed, e.g., https://twitter.com/trimetalerts

Examples:
https://transit.alerts.social/o/6447f3679b136eac97572961
https://transit.alerts.social/o/64483e679b136eac97572af1
https://transit.alerts.social/o/6448919c9b136eac97572d3d
https://transit.alerts.social/o/6448b8489b136eac97572dc5
https://transit.alerts.social/o/644551929b136eac97571e00

**Comment by kona314** [comment]
Similar to #8, the alerts the account shares are straight out of TriMet's GTFS-realtime alerts feed. There's likely a data issue on TriMet's side that causing internal information to appear publicly. I can follow up with them on this as well.
